src/main/java/frc/robot/commands/Auto/DriveSquareAuto.java

```diff
@@ -22,14 +24,22 @@ public DriveSquareAuto() {
 
     // Add your commands in the addCommands() call, e.g.
     // addCommands(new FooCommand(), new BarCommand());
-    addCommands(new DriveDistance(distance, speed,true),
+    addCommands(/**new DriveDistance(distance, speed,true),
                 new TurnAngle(angle, speed, true),
```

Comment:
Please remove older commented out code. We can use git commit history to refer to old changes if we want to.